config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   root "articles#index"
-  resources :articles
+  resources :articles do
+    resources :comments
```

Comment:
What is done on this line?

Reply:
This is Resourceful Routing. This permits that with a single call to resources can declare all of the necessary routes for `index`, `show`, `new`, `edit`, `create`, `update`, and `destroy` actions

Reply:
This is possible when we have resources that are logically children of other resources. In this way, this will also create routing helpers. However, the deep-nested resources quickly become cumbersome. Ruby on Rails recommends that resources should never be nested more than 1 level deep.